core/templates/services/state-interaction-stats.service.ts

```diff
@@ -34,34 +32,26 @@ import { UrlInterpolationService } from
   'domain/utilities/url-interpolation.service';
 
 export interface IAnswerData {
-  /* eslint-disable camelcase */
-  answer; // Type depends on interaction id.
-  frequency: number;
-  is_addressed?: boolean;
-  /* eslint-enable camelcase */
+  'answer': any; // Needs to be 'any' because type depends on interaction id.
+  'frequency': number;
+  'is_addressed'?: boolean;
```

Comment:
Why is this an optional field? Can we make it mandatory?

Reply:
Hmm I don't think so, if addressed info isn't supported then this should be undefined or null since letting it be false in such a case is semantically wrong.

Reply:
Ah ok, can you add a comment above it explaining the cases where it is not present (maybe this happens for specific interactions, I'm not sure)?

Reply:
Done.

Reply:
Sorry -- please could you also give some examples, in the comment, of visualizations which can present addressed answers, and some which can't, for clarity? (I'm reasonably familiar with this part of the codebase but I am still not actually sure what exactly is meant here.)